src/Slices/ServiceInventory/UI/ServiceInventory.test.tsx

```diff
@@ -498,7 +500,7 @@ test("GIVEN ServiceInventory WHEN sorting changes AND we are not on the first pa
 
   //sort on the second page
   await act(async () => {
-    await userEvent.click(screen.getByRole("button", { name: "State" }));
+    await userEvent.click(screen.getAllByRole("button", { name: "State" })[0]);
```

Comment:
How comes there are suddenly multiple buttons with the same aria-label?

Reply:
During the development I got out of the blue error that there is multiple elements with that name, and the correct one was 2 in the array. but at the end it was failing again, so I thought it got somehow reordered, I don't know what happened here, as I wasn't changing anything there, but it got somehow to back to the original way - changed back to what it was